packages/auth-construct/src/construct.ts

```diff
@@ -214,6 +218,37 @@ export class AmplifyAuth
     );
   }
 
+  /**
+   * Define bindCustomAttribute to meet requirements of the Cognito API to call the bind method
+   */
+  public bindCustomAttribute = (
+    key: string,
+    attribute: CustomAttributes
+  ): CustomAttributeConfig & ICustomAttribute => {
+    const config: CustomAttributeConfig = {
+      dataType: attribute.dataType,
+      mutable: attribute.mutable ?? true,
+      stringConstraints:
```

Comment:
if `attribute.dataType === 'Number'`, do we still want the undefined `stringConstraints` to be there?

If not, we should conditionally add `stringConstraints` and `numberConstraints`

Reply:
Updated.